google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -487,7 +645,14 @@ public BulkMutationBatcher newBulkMutationBatcher() {
    *         .setCell("[FAMILY]", "[QUALIFIER]", "[VALUE]")
    *       );
    *
-   *   boolean success = bigtableClient.checkAndMutateRowCallable().call(mutation);
+   *   // Sync style
+   *   try {
+   *     boolean success = bigtableClient.checkAndMutateRowCallable().call(mutation);
+   *   } catch (NotFoundException e) {
+   *     System.out.println("Tried to mutate a non-existent table");
```

Comment:
row doesn't exist or table doesn't?

Reply:
table, I extended the example to show how unmatched rows behave

Reply:
table